package.json

```diff
@@ -25,6 +25,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
     "react-test-renderer": "^16.9.0",
+    "redux-saga": "~1.1.3",
```

Comment:
@krizic why did you add `redux-saga` as a dependency to the monorepo?

Reply:
was failing on commit hook - (using node v10.17.0 (npm v6.11.3))

Reply:
removed, since it didn't belong to this issue